package.json

```diff
@@ -18,6 +18,7 @@
     "check-licenses": "ts-node scripts/check-licenses.ts",
     "release-apps": "ts-node scripts/release-apps.ts",
     "release-components": "ts-node scripts/release-components.ts",
+    "dev:vscode": "cd ./integrations/vscode && pnpm run dev",
```

Comment:
Currently the `dev` script runs `dev:build` which runs `vsce package` and installs the extension (see https://github.com/fwouts/previewjs/blob/4b5758873dddc089dad9da1fec457dd2dbb6eca3/integrations/vscode/package.json#L95), I feel like we don't need to do at least the install anymore, correct?

Reply:
You don't need to install the extension in the Extension Development Host because it's taken care of by `--extensionDevelopmentPath` flag, see https://github.com/microsoft/vscode-extension-samples/blob/main/helloworld-sample/.vscode/launch.json#L13 for example.

The problem is that the path had to be updated to be `${workspaceFolder}/integrations/vscode` so it knows where to find it :)

I made that change in https://github.com/fwouts/previewjs/commit/9919e32ca28c85147caf2bd5c60809051f1d4d6f. You'll notice that Preview.js doesn't appear in the list of installed extensions, but it's working nonetheless.